Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -708,13 +729,39 @@ public void OnInputDown(InputEventData eventData)
                 {
                     StartInputTimer(true);
                     SetPress(true);
+                    eventData.Use();
                 }
             }
         }
 
         public void OnInputPressed(InputEventData<float> eventData)
         {
+            /* MRTK Public Update ***************************/
             // ignore
+
+            /* MRTK Internal Update ***************************
```

Comment:
This should be fixed. It's unclear what these comments are for. It looks like this is cruft from working in two code bases.

Reply:
Added a TODO with a work pointer, I will find the person to do this.